data/Makefile

```diff
@@ -690,20 +690,20 @@ minsize=2000
 
 # Determine the maximum spanning tree.
 %.mst.tsv: %.tsv
-	$(python) $(bin)/physlr mst -V$V --prune=10 $< >$@
+	$(python) $(bin)/physlr mst -V$V --prune=10 --prune-bridges=10 $< >$@
```

Comment:
If `--prune` is for branches, and `--prune-bridges` is for bridges, should we also rename `--prune` to `--prune-branches` to make its purpose clear?

Reply:
Also, should we we add these values as variables in the Makefile so they are easier to change?

Reply:
makes sense! I'll just do it.

Reply:
@lcoombe I was checking the Makefile and I found that most of the params are not defined as variables. Also, we use two different values of `prune-branches` one for `backbone` (0 now) and the other for `backbone-graph` (10 now). Should I define 2 different Makefile variables for these two stages? alternatively I can embed a fixed value (0) for `backbone`.
Or I can forget about adding Makefile variables for them. Adding them makes my troubleshooting easier but ... yeah

Reply:
I'd say if values are hard-coded right now (ie we won't change the values usually), it would be fine to keep as is, but if you will be changing the values I think it's worth adding to the Makefile vs. having to go in and edit the value each time.
So yeah, we could add variables as we do PRs like this -- so only make the values variables when we realize we want to change them?

Reply:
So I am done with `--prune-branches`, and we only need to decide about the Makefile

Reply:
I'm fine with it either way -- Just thought it would be easier for you to troubleshoot if you're going to be changing that parameter, but it's totally up to you!

Reply:
sorry it did not show me your recent comment. Yeah I think it's good to stay with harcoded version for now; I don't think we will need to change it for different runs. So no more changes and I just added a 3rd one: `--prune-junctions`

Reply:
Sounds good!